### PR TITLE
[DOCS] Update Configuring Data Docs documentation for V3 API CLI

### PR DESCRIPTION
**Pull request by anthonyburdi**
Changes proposed in this pull request:
- Configuring Data Docs documentation now references the v3 API CLI
- Note that there is potentially a bug with rendering Expectation comments. I've confirmed that they are saved in the expectation suite however they are not rendered in Data Docs as expected. See issue here #2674

### CODE REVIEW

File: docs/guides/how_to_guides/configuring_data_docs/how_to_add_comments_to_expectations_and_display_them_in_data_docs.rst [hunk 1]
```diff
@@ -13,99 +13,200 @@ This guide will help you add descriptive comments (or notes, here used interchan
...
+                )
 
+            .. image:: single_line_markdown_red.png
+                :width: 800
+                :alt: Expectation with a single line of markdown comment is rendered in red with markdown formatting.
 
-    Multiple lines can be rendered by using a list for ``content``, these lines are rendered in black text with any Markdown formatting applied.
 
-    .. code-block:: python
+            Multiple lines can be rendered by using a list for ``content``, these lines are rendered in black text with any Markdown formatting applied.
```
Comment (talagluck) [inline docs/guides/how_to_guides/configuring_data_docs/how_to_add_comments_to_expectations_and_display_them_in_data_docs.rst]:
```suggestion
            Multiple lines can be rendered by using a list for ``content``; these lines are rendered in black text with any Markdown formatting applied.
```

File: docs/guides/how_to_guides/configuring_data_docs/how_to_add_comments_to_expectations_and_display_them_in_data_docs.rst [hunk 2]
```diff
@@ -13,99 +13,200 @@ This guide will help you add descriptive comments (or notes, here used interchan
...
-        :alt: Multiple lines of string rendered without formatting.
-    
+                batch.expect_column_values_to_not_be_null(
+                    column="column_name",
+                    meta={
+                        "notes": {
+                            "format": "string",
+                            "content": [
+                                "Example notes about this expectation. **Markdown** `Supported`.",
+                                "Second example note **with** *Markdown*",
```
Comment (talagluck) [inline docs/guides/how_to_guides/configuring_data_docs/how_to_add_comments_to_expectations_and_display_them_in_data_docs.rst]: Very, very tiny nit, but should the text say `Not Supported` and `**without** *Markdown*`? (I realize this isn't new code, but figured I would raise it).

File: docs/guides/how_to_guides/configuring_data_docs/how_to_add_comments_to_expectations_and_display_them_in_data_docs.rst [hunk 3]
```diff
@@ -13,99 +13,200 @@ This guide will help you add descriptive comments (or notes, here used interchan
...
+            .. code-block:: python
+
+                validator.expect_column_values_to_not_be_null(
+                    column="column_name",
+                    meta={
+                        "notes": {
+                            "format": "string",
+                            "content": [
+                                "Example notes about this expectation. **Markdown** `Supported`.",
+                                "Second example note **with** *Markdown*",
```
Comment (talagluck) [inline docs/guides/how_to_guides/configuring_data_docs/how_to_add_comments_to_expectations_and_display_them_in_data_docs.rst]: See above

File: docs/guides/how_to_guides/configuring_data_docs/how_to_add_comments_to_expectations_and_display_them_in_data_docs.rst [hunk 4]
```diff
@@ -13,99 +13,200 @@ This guide will help you add descriptive comments (or notes, here used interchan
...
+                        }
+                    }
+                )
+
+            .. image:: single_line_markdown_red.png
+                :width: 800
+                :alt: Expectation with a single line of markdown comment is rendered in red with markdown formatting.
+
+
+            Multiple lines can be rendered by using a list for ``content``, these lines are rendered in black text with any Markdown formatting applied.
```
Comment (talagluck) [inline docs/guides/how_to_guides/configuring_data_docs/how_to_add_comments_to_expectations_and_display_them_in_data_docs.rst]:
```suggestion
            Multiple lines can be rendered by using a list for ``content``; these lines are rendered in black text with any Markdown formatting applied.
```

File: docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_a_filesystem.rst [hunk 5]
```diff
@@ -29,21 +29,46 @@ Steps
 
 2. **Test that your configuration is correct by building the site.**
 
-  Use the following CLI command: ``great_expectations docs build --site-name local_site``. If successful, the CLI will open your newly built Data Docs site and provide the path to the index page.
+.. content-tabs::
```
Comment (talagluck) [inline docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_a_filesystem.rst]: This is awesome! The content tabs not being at the very top makes sense, but I wonder if there might be confusion about which API these docs are for? (This is less of an issue here, where the tabs are in the second step, and more of an issue with the GCS docs, where the tabs are a lot lower).

I guess I'm wondering if it's worth putting the content tabs at the top, and duplicating the code between tabs, or maybe putting a note at the top that the initial steps are the same, and when the paths diverge we'll have explanations? I don't love either of these, and might be overthinking, but wanted to raise.

File: docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.rst [hunk 6]
```diff
@@ -120,21 +120,47 @@ This guide will explain how to host and share Data Docs on Google Cloud Storage.
 
 7. **Build the GCS Data Docs site.**
 
-  Use the following CLI command: ``great_expectations docs build --site-name gs_site``. If successful, the CLI will provide the object URL of the index page. Since the bucket is not public, this URL will be inaccessible. Rather, you will access the Data Docs site using the App Engine app configured above.
+.. content-tabs::
```
Comment (talagluck) [inline docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.rst]: See note on tab location above

File: docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_s3.rst [hunk 7]
```diff
@@ -91,21 +91,45 @@ Steps
 
 5. **Test that your configuration is correct by building the site.**
 
-  Use the following CLI command: ``great_expectations docs build --site-name s3_site``. If successful, the CLI will open your newly built S3 Data Docs site and provide the URL, which you can share as desired. Note that the URL will only be viewable by users with IP addresses appearing in the above policy.
+.. content-tabs::
```
Comment (talagluck) [inline docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_s3.rst]: See note on tab location above.

**Review by talagluck — APPROVED** [review]
These are awesome! I have a few very minor nits, and some thoughts about content tab positioning, but none of these are blocking, so I'm going to approve.

File: docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_a_filesystem.rst (hunk 5 above)
Comment (anthonyburdi) [reply docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_a_filesystem.rst]: Great call! I think you are right to just put them at the top - I was going for less code duplication but it makes sense in docs and I haven't yet found an easy way to reference a code snippet in another section of the docs.

File: docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.rst (hunk 6 above)
Comment (anthonyburdi) [reply docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.rst]: Done!

File: docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_s3.rst (hunk 7 above)
Comment (anthonyburdi) [reply docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_s3.rst]: Done!

**Review by talagluck — APPROVED** [review]
LEBTM (Looks Even Better To Me)!

**Pull request merged**